### PR TITLE
Updates for managed cluster labels variable

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-4543

### CODE REVIEW

**Review by JustinKuli — APPROVED** [review]

**Review by JustinKuli — APPROVED** [review]

File: governance/template_functions.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
+Template functions, such as resource-specific and generic `lookup` template functions, are available for referencing Kubernetes resources on the hub cluster (using the `{{hub ... hub}}` delimiters), or managed cluster (using the `{{ ... }}` delimiters). See _Template processing_ for more details. The resource-specific functions are used for convenience and makes content of the resources more accessible. If you use the generic function, `lookup`, which is more advanced, it is best to be familiar with the YAML structure of the resource that is being looked up. In addition to these functions, utility functions such as `base64encode`, `base64decode`, `indent`, `autoindent`, `toInt`, `toBool`, and more are also available.
```
Comment (cadawson) [inline governance/template_functions.adoc]:
```suggestion
Template functions, such as resource-specific and generic `lookup` template functions, are available for referencing Kubernetes resources on the hub cluster (using the `{{hub ... hub}}` delimiters), or on the managed cluster (using the `{{ ... }}` delimiters). See _Template processing_ for more details. The resource-specific functions are used for convenience and makes content of the resources more accessible. If you use the generic function, `lookup`, which is more advanced, familiarize yourself with the YAML structure of the resource that is being looked up. In addition to these functions, utility functions such as `base64encode`, `base64decode`, `indent`, `autoindent`, `toInt`, `toBool`, and more are also available.
```

**Review by cadawson — APPROVED** [review]
Suggestion, but looks good. Thanks!

File: governance/template_functions.adoc (hunk 1 above)
Comment (dockerymick) [reply governance/template_functions.adoc]: Great suggestions! TEAM ACTIVE TONE !

**Comment by dockerymick** [comment]
@cadawson thanks for your suggestions and approvals! I have accepted them. Merging this PR

**Pull request merged**